selfdrive/car/subaru/interface.py

```diff
@@ -10,47 +10,45 @@ class CarInterface(CarInterfaceBase):
 
   @staticmethod
   def _get_params(ret, candidate: CAR, fingerprint, car_fw, experimental_long, docs):
-    platform_flags = candidate.config.flags
-
```

Comment:
Yea I forgot that the flags will be set by this point, LGTM

Reply:
I'm wondering if we really want to distinguish between static/dynamic. For example in Chrysler we set the higher min steering speed based on platform and on eps fw version, so really this could both be a static or dynamic flag, and I think we want to be moving away from the static flags in general (ex: HYBRID)

Reply:
I think we should treat it as flags that are allowed to be statically set in values.py, there are flags we don't ever expect to be set there and may break things because of that expectations.

Reply:
I agree with moving away from static flags, but we will need them for some time until we can detect everything, plus they help with the car docs.